docs/src/main/asciidoc/spring-cloud-netflix.adoc

```diff
@@ -723,6 +723,39 @@ don't want to use Eureka, you can simply configure a list of servers
 in your external configuration (see
 <<spring-cloud-ribbon-without-eureka,above for example>>).
 
+Feign is capable of inspecting whole interface hierarchies allowing to
+group the common operations into convenient base interfaces.
+Together with Spring MVC you can share the same contract for your
+REST endpoint and Feign client.
```

Comment:
"whole interface hierarchies" is to broad.  I think we need to mention the [rules](https://github.com/Netflix/feign/issues/133#issuecomment-118033442) from feign.